src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -300,6 +303,63 @@ private void testAddQuestionAction() throws Exception {
         feedbackEditPage.verifyHtmlMainContent("/instructorFeedbackQuestionAddSuccess.html");
     }
 
+    private void testAddTemplateQuestionLink() throws Exception {
+
+        ______TS("add template question link");
+
+        feedbackEditPage.clickNewQuestionButton();
+        feedbackEditPage.clickTemplateModalButton();
+
+        assertFalse("Unable to submit when there is no question selected",
+                feedbackEditPage.isAddTemplateQuestionButtonEnabled());
+    }
+
+    private void testAddTemplateQuestionAction() throws Exception {
+
+        ______TS("add template question action success");
+
+        // check if each question panel is expandable
+        feedbackEditPage.clickTemplateQuestionPanel(1);
+        feedbackEditPage.clickTemplateQuestionPanel(2);
+        feedbackEditPage.clickTemplateQuestionPanel(3);
+        feedbackEditPage.clickTemplateQuestionPanel(4);
+        feedbackEditPage.clickTemplateQuestionPanel(5);
+        feedbackEditPage.verifyHtmlPart(By.id("addTemplateQuestionModal"),
+                "/instructorFeedbackTemplateQuestionExpandedModal.html");
+
+        feedbackEditPage.clickTemplateQuestionModalCheckBox(1);
+
+        assertTrue("Can click after selecting a question",
+                feedbackEditPage.isAddTemplateQuestionButtonEnabled());
+        feedbackEditPage.clickAddTemplateQuestionButton();
+
+        feedbackEditPage.waitForTextsForAllStatusMessagesToUserEquals(Const.StatusMessages.FEEDBACK_QUESTION_ADDED);
+        assertNotNull(getFeedbackQuestionWithRetry(courseId, feedbackSessionName, 2));
+
+        feedbackEditPage.verifyHtmlMainContent("/instructorFeedbackTemplateQuestionAddSuccess.html");
+
+        ______TS("add multiple template questions success");
+
+        feedbackEditPage.clickNewQuestionButton();
+        feedbackEditPage.clickTemplateModalButton();
+
+        feedbackEditPage.clickTemplateQuestionModalCheckBox(2);
+        feedbackEditPage.clickTemplateQuestionModalCheckBox(3);
+        feedbackEditPage.clickAddTemplateQuestionButton();
+
+        feedbackEditPage.waitForTextsForAllStatusMessagesToUserEquals(
+                Const.StatusMessages.FEEDBACK_QUESTION_ADDED_MULTIPLE);
+        assertNotNull(getFeedbackQuestionWithRetry(courseId, feedbackSessionName, 4));
+
+        // revert back to state expected by tests after this by deleting newly added template questions
+        String questionId = getFeedbackQuestionWithRetry(courseId, feedbackSessionName, 4).getId();
+        BackDoor.deleteFeedbackQuestion(questionId);
+        questionId = getFeedbackQuestionWithRetry(courseId, feedbackSessionName, 3).getId();
+        BackDoor.deleteFeedbackQuestion(questionId);
+        questionId = getFeedbackQuestionWithRetry(courseId, feedbackSessionName, 2).getId();
+        BackDoor.deleteFeedbackQuestion(questionId);
+    }
+
```

Comment:
Lets add one more case for `Cancel` ?

Reply:
As copy modal currently does not have a `cancel` test case so I am not sure if template modal is necessary? 

Currently, there is no id specified for the modal cancel button so I will have to add an id to it and run `GodMode` again

Reply:
Hmm... then not necessary. Its not a required test case so its fine to let go if it will cause a lot of unnecessary changes